packages/jest-haste-map/src/haste_fs.js

```diff
@@ -44,6 +44,10 @@ export default class HasteFS {
     return Array.from(this._files.keys());
   }
 
+  getFileIterator(): Iterator<string> {
```

Comment:
@rafeca you'll be able to use this in Metro. It should be way faster if you just want to iterate over the files.